### ISSUE TITLE
Workspace does not work on mobile

### ISSUE THREAD

**Issue by Trenobol** [mirnovov/obsidian-homepage#91]
### Details

Workspace does not work on mobile

Notebooks and notes are not shown when I select Workspace

![1720626959733](https://github.com/mirnovov/obsidian-homepage/assets/118353130/6674b5db-0a66-4272-bc29-98c7e3cda040)
![IMG_5605](https://github.com/mirnovov/obsidian-homepage/assets/118353130/d36b1aa8-79c1-4784-90b8-1947a2422a84)

### Checklist

- [X] I updated to the latest version of the plugin.

**Comment by mirnovov** [comment]
Works for me. Does this occur when opening the workspace without Homepage?

**Comment by Cito** [comment]
Can confirm, same problem here. When I switch off the Homepage plugin, it works. I'm using the Workspace option, too.

**Comment by mirnovov** [comment]
@Cito what happens when opening the same workspace manually (without Homepage?). At least in my vault, I still get the same behaviour in that scenario, suggesting that this is not a problem with Homepage in particular.

**Comment by Cito** [comment]
@mirnovov You're right, the same things happens when I open it manually. It seems to be a more fundamental problem with using workspaces from desktop on mobile. Sorry for the noise.
